exec/java-exec/src/main/java/org/apache/drill/exec/work/foreman/Foreman.java

```diff
@@ -828,8 +828,13 @@ public void close() throws Exception {
         queryManager.writeFinalProfile(uex);
       }
 
-      // Remove the Foreman from the running query list.
-      bee.retireForeman(Foreman.this);
+      try {
+        queryContext.close();
+      } catch (Exception e) {
+        final String message = String.format("Unable to close query context for query {}",
+          QueryIdHelper.getQueryId(queryId));
+        logger.error(message, e);
```

Comment:
```
logger.error("Unable...query {}", QueryIdHelper...);
```

The `{}` is a logger format pattern, not a `String.format()` pattern.

Reply:
Done